### ISSUE TITLE
a11y: Buttons should announce that they are a button

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#11992]
They should also have the double tap to activate hint.

**Comment by amirh** [comment]
The double tap to activate hint is now tracked separately as part of https://github.com/flutter/flutter/issues/12636
